src/components/bs5/footer/footer.functions.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Init footer formio feedback form
+ *
+ * @memberof module:Footer
+ *
+ * @returns {void}
+ */
+export function displayFeedbackForm() {
+  // eslint-disable-next-line no-undef
+  FormioScript.init();
```

Comment:
For technical debt removal:

move formio loader script to data attribute (make it optional)

wrap .init() in a check to see if formioscript is already loaded on the page.
if not existing, check data element on feedback form (if set)
else use hard coded url in this footer.functions.js for formio loader.
load the javascript and fire off the init function.